src/TokenHandler/AccountManager.mo

```diff
@@ -71,25 +83,61 @@ module {
     /// Retrieves the current fee amount.
     public func fee() : Nat = fee_;
 
-    /// Retrieves the allowed minimal deposit.
-    public func minimum() : Nat = depositRegistry.minimum();
+    /// Retrieves the admin-defined deposit minimum.
+    public func definedDepositMinimum() : Nat = definedDepositMinimum_;
```

Comment:
Suggestion to reduce the number of public functions and to remove code repetition. We can have an enum argument `{ #deposit; #withdrawal }` and one set of functions instead of two sets that are called like this: 
```
setMinimum(#deposit, new_minimum);
definedMinimum(#deposit);
minimum(#deposit);
```

Reply:
Implemented